modules/apps/digital-signature/digital-signature-impl/src/main/java/com/liferay/digital/signature/internal/configuration/DigitalSignatureConfigurationDDMFormDeclaration.java

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation: